### PR TITLE
Align with upstream

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: Source/Core/DolphinQt/Main.cpp [hunk 1]
```diff
@@ -258,6 +258,7 @@ int main(int argc, char* argv[])
     Settings::Instance().InitDefaultPalette();
     Settings::Instance().UpdateSystemDark();
     Settings::Instance().SetCurrentUserStyle(Settings::Instance().GetCurrentUserStyle());
+    //TODO: Settings::Instance().ApplyStyle();
```
[the remark this answers is not in the document]
Comment (SimoneN64) [reply Source/Core/DolphinQt/Main.cpp]: Yes, in fact it substitutes the line above

**Comment by SimoneN64** [comment]
Any updates on this?

**Comment by ximk** [comment]
I think we should realign again, and then review. Sorry for the wait.

**Comment by ghost** [comment]
I tried my best... We'll need testing

**Pull request closed without merging**